tools/hidl/hidlcomp.cpp

```diff
@@ -3153,6 +3153,9 @@ void EspMessageInfo::write_esp_ipp()
         outf("\n\tC%s(IEspContext* ctx, const char *serviceName, IProperties *params, MapStrToBuf *attachments);", name_);
     }   
 
+    if (espm_type_==espm_request)
+        outs("\n\tIEspClientRpcSettings &rpc(){return *static_cast<IEspClientRpcSettings*>(this);}\n\n");
```

Comment:
Is this generating the implementation of a method called rpc? 
I didn't see that method called in any of the new code. Also, is there a more descriptive name?